ArduCopter/land_detector.cpp

```diff
@@ -65,7 +65,8 @@ void Copter::update_land_detector()
         // because multi's use throttle), check that collective pitch is below land min collective position or throttle stick is zero.
         // Including the throttle zero check will ensure the conditions where stabilize stick zero position was not below collective min. For modes
         // that use altitude hold, check that the pilot is commanding a descent and collective is at min allowed for altitude hold modes.
-        bool motor_at_lower_limit = ((flightmode->has_manual_throttle() && (motors->get_below_land_min_coll() || heli_flags.coll_stk_low) &&
+        bool motor_at_lower_limit = (!force_flying || landing) &&
```

Comment:
landing variable has not been defined.  Otherwise, I think this accomplishes the intended function.